log4j-1.2-api/src/test/java/org/apache/log4j/config/RewriteAppenderTest.java

```diff
@@ -73,5 +74,6 @@ public void testRewrite() throws Exception {
         assertNotNull("No properties in the event", events.get(0).getProperties());
         assertTrue("Key was not inserted", events.get(0).getProperties().containsKey("key2"));
         assertEquals("Key value is incorrect", "Log4j", events.get(0).getProperties().get("key2"));
+        assertTrue("Timestamp is before point of logging", events.get(0).getTimeStamp() >= logTime);
     }
```

Comment:
I'm not sure if we've fixed this elsewhere but I've actually seen this kind of test fail when running builds just around the daylight saving time change. Probably not worth making the test more complex.

Reply:
The milliseconds since epoch which the timestamp represents are not influenced by timezones and daylight saving time.

Reply:
That's correct, this should only be able to fail if the system clock is updated mid-test, which I wouldn't expect.